.github/workflows/run_tests_uds.yml

```diff
@@ -0,0 +1,146 @@
+name: run_tests
```

Comment:
Because we're copying files to the local system and executing them that way -- which would conflict with the way that files are copied to the local system for the Redis clustered tests.